### DISCUSSION TITLE
The EMBEDDING_ID variable is the id of the embedding which you can get from embedstore.ai

### DISCUSSION THREAD

**Discussion by AminaFaragai** [mem0ai/mem0#803] [Q&A]
Please where can i get the embedding id? And what does it look like? Following a tutorial and he gave this link to the id but I didn't know where the id is.

**Comment by deshraj** [comment]
Hey, sorry but I dont think your question is clear to me. Can you please elaborate more? Also, which tutorial were you following?

**Comment by taranjeet** [comment]
closing this discussion as it is inactive for long.
